api/logger/log.config

```diff
@@ -0,0 +1,9 @@
+{
+  "handlerTypes": ["none"],
+  "level": "DEBUG",
+  "format": "%(asctime)s %(levelname)s [%(filename)s:%(lineno)s - %(funcName)s()] %(message)s",
+  "appendToFile": true,
+  "filePath": "logger/logger.log",
```

Comment:
Could you log in the path "logs/nlp_logs.log", because this is where we will take the logs outside of the container through a volume to read logs on AWS. We don't want to take the logger code outside the container, only the logs.

Reply:
done